docs/visual-basic/programming-guide/language-features/data-types/tuples.md

```diff
@@ -41,6 +41,35 @@ Rather than using default names for a tuple's fields, you can instantiate a *nam
 
 [!code-vb[Instantiate](../../../../../samples/snippets/visualbasic/programming-guide/language-features/data-types/tuple1.vb#4)]
 
+## Inferred tuple field names
+
+Starting with Visual Basic 15.3, Visual Basic can infer the names of tuple elements; you do not have to assign them explicitly. Inferred tuple names are useful when you initialize a tuple from a set of variables, and you want the tuple field name to be the same as the variable name. 
+
+The following example creates a `stateInfo` tuple that contains three explicitly named fields, `state`, `stateName`, and `capital`. Note that, in naming the fields, the tuple initialization statement simply assigns the named fields the values of the identically named variables.
+
+[!code-vb[ExplicitlyNamed](../../../../../samples/snippets/visualbasic/programming-guide/language-features/data-types/named-tuples/program.vb#1)]
+ 
+Because fields and variables have the same name, the Visual Basic compiler can infer the names of the fields, as the following example shows.
+
+[!code-vb[ExplicitlyNamed](../../../../../samples/snippets/visualbasic/programming-guide/language-features/data-types/named-tuples/program.vb#2)]
+
+To enable interred tuple field names, you must define the version of the Visual Basic compiler to use in your Visual Basic project (\*.vbproj) file: 
+
+```xml 
+<PropertyGroup> 
+  <LangVersion>15.3</LangVersion> 
+</PropertyGroup> 
+
+The version number can be any version of the Visual Basic compiler starting with 15.3.
+
+In some cases, the Visual Basic compiler cannot infer the tuple field name from the candidate name0, and the tuple field can only be referenced using its default name, such as `Item1`, `Item2`, etc. These include:
+
+- The candidate name is the same as teh name of a tuple member, such as `Item3`, `Rest`, or `ToString`.
+
+- The candidate name is duplicated in the tuple.
+ 
+When field name inference fails, Visual Basic does not generate a compiler error, and an exception is not thrown at runtime. Instead, tuple fields must be referenced by their predefined names, such as `Item1` and `Item2`. 
```

Comment:
Suggestion: "error, and an exception is not thrown" maybe should be "error, nor is an exception thrown".